src/CircleMenu/CircleMenuItem/CircleMenuItem.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * The CircleMenuItem.
+ *
+ * @class CircleMenuItem
+ * @extends React.Component
+ */
+export class CircleMenuItem extends React.Component {
+
+  _ref = null;
+
+  static propTypes = {
+    /**
+     * The duration of the animation in milliseconds. Pass 0 to avoid animation.
+     * Default is 300.
+     *
+     * @type {Number}
+     */
+    animationDuration: PropTypes.number,
+    /**
+     * The index of the current CircleMenuItem.
+     */
+    idx: PropTypes.number,
+    /**
+     * The radius of the CircleMenu.
+     *
+     * @type {Number}
+     */
+    radius: PropTypes.number,
+    /**
+     * The children of the CircleMenuItem. Should be just one Node.
+     *
+     * @type {Node}
+     */
+    children: PropTypes.node,
```

Comment:
Absolutely minor, but I recently discovered the possibilty of custom validators in typechecking, see [here](https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes). Maybe this could be useful to check the length of the passed props here?

Reply:
I'll might add this later as there are allready some more features i'd like to add to the circle-menu.